ESPixelStick/src/input/InputFPPRemotePlayFile.cpp

```diff
@@ -283,27 +289,49 @@ bool c_InputFPPRemotePlayFile::ParseFseqFile ()
                 CurrentSparseRange.DataOffset   = read24 (FseqRawRanges[SparseRangeIndex].Start);
                 CurrentSparseRange.ChannelCount = read24 (FseqRawRanges[SparseRangeIndex].Length);
                 TotalChannels += CurrentSparseRange.ChannelCount;
-                LargestBlock  = max (LargestBlock, CurrentSparseRange.ChannelCount);
```

Comment:
The offset is not going to effect the largest block as the blocks will also be indexed at 0